uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart2.tpl

```diff
@@ -35,7 +35,10 @@
             <button class="button__delete btn-warning">Disable User Account</button>
           </div>
           <div class="col-md-6">
-            <button class="button__disable btn-danger">Delete User Account</button>
+            <button class="button__disable btn-danger" 
```

Comment:
What is the role of `button__disable` or `button__delete` class?

Reply:
`button__disable` or `button__delete` are wrapper class for bootstrap `bt` class to customize the buttons to make as per the button design in contact action menu.

Reply:
Ok, so as I understand this is a custom class. Then all custom classes should have `crm_` prefix for admin section and `chr_` for SSP with no exceptions.
@AkA84 Could you confirm this please.